tests/helpers/test_intent.py

```diff
@@ -710,7 +710,7 @@ async def test_invalid_area_floor_names(hass: HomeAssistant) -> None:
             "TestType",
             slots={"area": {"value": "invalid area"}},
         )
-        assert err.value.result.no_match_reason == intent.MatchFailedReason.INVALID_AREA
+    assert err.value.result.no_match_reason == intent.MatchFailedReason.INVALID_AREA
```

Comment:
Hello @synesthesiam, can you look as this test? It fails but went unnoticed because the assert was never executed inside the pytest.raises block. Thanks!